flow/designs/gf12/ariane/rules-base.json

```diff
@@ -27,32 +27,32 @@
         },
         {
             "field": "cts__timing__setup__ws",
-            "value": -199.61,
+            "value": -314.11,
             "compare": ">="
         },
         {
             "field": "cts__timing__setup__ws__pre_repair",
-            "value": -584.05,
+            "value": -919.06,
             "compare": ">="
         },
         {
             "field": "cts__timing__setup__ws__post_repair",
-            "value": -583.61,
+            "value": -918.57,
             "compare": ">="
         },
         {
             "field": "globalroute__timing__clock__slack",
-            "value": 0.0,
+            "value": -108.41,
             "compare": ">="
         },
         {
             "field": "globalroute__timing__setup__ws",
-            "value": 0.0,
+            "value": -108.41,
```

Comment:
finish__timing__setup__ws is still zero - could this be ratcheted further?

Reply:
It can be but the global route timing will have larger negative slack. There is still a difference between global route numbers and OpenRCX based finish numbers. I am currently keeping the global route numbers close to 0.